modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -51,16 +51,16 @@ def wait_computations_if_benchmark_mode(func):
     -----
     `func` should return NumPy array with partitions.
     """
-    if BenchmarkMode.get():
-
-        @wraps(func)
```

Comment:
If BenchmarkMode is enabled here, shouldn't we be blocking? Am I missing something here?

Reply:
Resolved offline. We only evaluate the benchmark mode once, when we run `wait_computations_if_benchmark_mode`.